interop-cats/jvm/src/test/scala/scalaz/zio/interop/catzSpec.scala

```diff
@@ -142,6 +142,16 @@ class catzSpec
     def effect[R: Runtime]           = Effect[TaskR[R, ?]]
   }
 
+  object summoningRuntimeInstancesTest {
+    import cats.effect.{ Clock => CatzClock, _ }
+    import scalaz.zio.interop.catz.implicits._
+
+    ContextShift[Task]
+    ContextShift[TaskR[String, ?]]
+    CatzClock[Task]
+    Timer[Task]
+  }
```

Comment:
These are great. It occurs to me, users should be able to get them from `UIO`, too. Maybe worth adding a test here.